api/envoy/service/auth/v2alpha/attribute_context.proto

```diff
@@ -1,13 +1,16 @@
 syntax = "proto3";
 
-// [#proto-status: draft]
-
 package envoy.service.auth.v2alpha;
 
 import "envoy/api/v2/core/address.proto";
 
 import "google/protobuf/timestamp.proto";
 
+// [#protodoc-title: Attribute Context ]
+
+// See :ref:`network filter configuration overview <config_network_filters_ext_authz>`
```

Comment:
Please make sure this all renders correctly. I think this commend might need to be attached to the one on line 9 but I'm not sure. Same elsewhere.